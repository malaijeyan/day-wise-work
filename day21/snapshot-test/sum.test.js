import sum from "./sum";
test("snapshot testing-sum",()=>{
    expect(sum(4,5)).toMatchSnapshot();
    expect(sum(3,4)).toMatchSnapshot();
    expect(sum(2,2)).toMatchSnapshot();
    expect(sum(1,8)).toMatchSnapshot();
    expect(sum(-1,6)).toMatchSnapshot();
})










