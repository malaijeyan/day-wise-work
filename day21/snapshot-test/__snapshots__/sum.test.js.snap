// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing-sum 1`] = `9`;

exports[`snapshot testing-sum 2`] = `7`;

exports[`snapshot testing-sum 3`] = `4`;

exports[`snapshot testing-sum 4`] = `9`;

exports[`snapshot testing-sum 5`] = `5`;
